composer.json

```diff
@@ -34,15 +34,13 @@
     "phpstan/phpstan": "^1.10.47",
     "phpstan/phpstan-deprecation-rules": "^1.1",
     "phpstan/phpstan-doctrine": "^1.3.40",
-    "phpstan/phpstan-php-parser": "^1.1",
     "phpstan/phpstan-phpunit": "^1.3",
     "phpstan/phpstan-symfony": "^1.3",
     "phpunit/phpunit": "^9.5",
     "rector/rector": "^0.18.12",
     "symfony/browser-kit": "~5.4.0",
     "symfony/dom-crawler": "~5.4.0",
     "symfony/maker-bundle": "^1.38",
-    "symfony/phpunit-bridge": "~5.4.0",
```

Comment:
This is used by Symfony to be able to run PHPUnit 8/9/10 on different Symfony LTS and PHP version. Not really useful outside symfony/symfony monorepo, but projects keep copying it.

We use single PHP version here, so we can drop it.